src/Symfony/Bridge/Twig/Tests/Command/LintCommandTest.php

```diff
@@ -107,6 +108,44 @@ public function testLintDefaultPaths()
         self::assertStringContainsString('OK in', trim($tester->getDisplay()));
     }
 
+    public function testLintIncorrectFileWithGithubFormat()
+    {
+        if (!class_exists(GithubActionReporter::class)) {
+            $this->expectException(\InvalidArgumentException::class);
+            $this->expectExceptionMessage('The "github" format is only available since "symfony/console" >= 5.3');
+        }
+
+        $filename = $this->createFile('{{ foo');
+        $tester = $this->createCommandTester();
+
+        $tester->execute(['filename' => [$filename], '--format' => 'github'], ['decorated' => false]);
+
+        if (!class_exists(GithubActionReporter::class)) {
+            return;
+        }
+
+        self::assertEquals(1, $tester->getStatusCode(), 'Returns 1 in case of error');
+        self::assertStringMatchesFormat('%A::error file=%s, line=1, col=0::Unexpected token "end of template" ("end of print statement" expected).%A', trim($tester->getDisplay()));
```

Comment:
```suggestion
        self::assertStringMatchesFormat('%A::error file=%s,line=1,col=0::Unexpected token "end of template" ("end of print statement" expected).%A', trim($tester->getDisplay()));
```

extra spaces were removed in https://github.com/symfony/symfony/commit/bd638a0fd81b3e2a652cf221d427015e7a0c4ff7, that's why your tests are failing actually I guess, not because of a missing `GithubActionReporter `